src/prob/tp_opf.jl

```diff
@@ -55,5 +59,18 @@ function post_tp_opf(pm::GenericPowerModel)
         PMs.constraint_dcline(pm, i, cnd=c)
     end
 
+    if haskey(ref(pm), :trans)
```

Comment:
Trans `trans` should be required in the data model (even if empty) so that check can be avoided.

Reply:
I added this check at the end of the OpenDSS parser. I also added this check in add_arcs_trans!, because otherwise data being parsed from the matlab format would not contain the trans key.